aws-opentelemetry-distro/src/amazon/opentelemetry/distro/aws_opentelemetry_configurator.py

```diff
@@ -263,23 +274,30 @@ def _customize_sampler(sampler: Sampler) -> Sampler:
 
 
 def _customize_exporter(span_exporter: SpanExporter, resource: Resource) -> SpanExporter:
-    if not _is_application_signals_enabled():
-        return span_exporter
     if _is_lambda_environment():
+        # TODO: not override span exporter if set OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
+        # TODO: not override span exporter if set console exporter
         traces_endpoint = os.environ.get(AWS_XRAY_DAEMON_ADDRESS_CONFIG, "127.0.0.1:2000")
-        return AwsMetricAttributesSpanExporterBuilder(OTLPUdpSpanExporter(endpoint=traces_endpoint), resource).build()
+        span_exporter = OTLPUdpSpanExporter(endpoint=traces_endpoint)
+
+    if not _is_application_signals_enabled():
```

Comment:
is it better to disable AppSignals metrics by not adding `AwsSpanMetricsProcessor`?
```
# Construct and set application signals metrics processor
provider.add_span_processor(AwsSpanMetricsProcessorBuilder(meter_provider, resource).build())
```